specification/purview/resource-manager/Microsoft.Purview/stable/2021-12-01/purview.json

```diff
@@ -0,0 +1,2685 @@
+          "readOnly": true
+        }
+      }
+    },
+    "PrivateEndpointConnectionProperties": {
+      "description": "A private endpoint connection properties class.",
+      "type": "object",
+      "properties": {
+        "privateEndpoint": {
+          "$ref": "#/definitions/PrivateEndpoint",
+          "description": "The private endpoint information."
+        },
+        "privateLinkServiceConnectionState": {
+          "$ref": "#/definitions/PrivateLinkServiceConnectionState",
+          "description": "The private link service connection state."
+        },
+        "provisioningState": {
+          "description": "The provisioning state.",
+          "type": "string",
+          "readOnly": true
+        }
+      }
+    },
+    "PrivateEndpoint": {
+      "description": "A private endpoint class.",
+      "type": "object",
+      "properties": {
+        "id": {
+          "description": "The private endpoint identifier.",
+          "type": "string"
+        }
+      }
+    },
+    "PrivateLinkServiceConnectionState": {
+      "description": "The private link service connection state.",
+      "type": "object",
+      "properties": {
+        "actionsRequired": {
+          "description": "The required actions.",
+          "type": "string"
+        },
+        "description": {
+          "description": "The description.",
+          "type": "string"
+        },
+        "status": {
+          "description": "The status.",
+          "enum": [
+            "Unknown",
+            "Pending",
+            "Approved",
+            "Rejected",
+            "Disconnected"
+          ],
+          "type": "string",
+          "x-ms-enum": {
+            "name": "status",
+            "modelAsString": true
+          }
+        }
+      }
+    },
+    "ErrorResponseModel": {
+      "description": "Default error response model",
+      "type": "object",
+      "properties": {
+        "error": {
+          "description": "Gets or sets the error.",
+          "allOf": [
+            {
+              "$ref": "#/definitions/ErrorModel"
+            }
+          ],
+          "readOnly": true
+        }
+      }
+    },
+    "AccountUpdateParameters": {
+      "description": "The account update properties.",
+      "type": "object",
+      "properties": {
+        "identity": {
+          "$ref": "#/definitions/Identity",
+          "description": "Identity related info to add/remove userAssignedIdentities."
+        },
+        "properties": {
+          "$ref": "#/definitions/AccountProperties",
+          "description": "The account properties."
+        },
+        "tags": {
+          "description": "Tags on the azure resource.",
+          "type": "object",
+          "additionalProperties": {
+            "type": "string"
+          }
+        }
+      }
+    },
+    "AccessKeys": {
+      "description": "The Account access keys.",
+      "type": "object",
+      "properties": {
+        "atlasKafkaPrimaryEndpoint": {
```

Comment:
Just confirming: is it a secret/credential?

Reply:
These are EH connection strings. We need to expose them for customer to perform ingestion.

Reply:
Got it, then it should add "x-ms-secret: true".

refs: https://github.com/Azure/autorest/blob/main/docs/extensions/readme.md#x-ms-secret